src/entities/events/ui/event-card/event-card.tsx

```diff
@@ -25,20 +25,20 @@ export const EventCard = ({
       : `• ${date} • ${time} • ${type}`;
 
   return (
-    <article className={cx('event-card')}>
-      <div className={cx('event-card__header')}>
+    <article className={cx('event-card')} data-testid="event-card">
+      <div className={cx('event-card__header')} data-testid="card-header">
```

Comment:
I suggest to use `kebab-case` to make our code base more consistent.

Reply:
Done